cmd/apps/openfaas_app.go

```diff
@@ -137,9 +137,7 @@ func MakeInstallOpenFaaS() *cobra.Command {
 
 		chartPath := path.Join(os.TempDir(), "charts")
 
-		println("foo")
```

Comment:
Hmm? :)

Reply:
yeh, my last PR had those in. :man_facepalming: :-1: